branding_guidelines.md

```diff
@@ -0,0 +1,90 @@
+| :heavy_check_mark: Use the provided original logos | :x: Create your own versions of the provided original logos |                             |
+| :heavy_check_mark: Use the original colors                  |               :x: Use our logos (for your own use-case) without permission
+| :heavy_check_mark: Use our logos commercially      |               :x: Sell our logos (for your own use-case) without permission
+| :heavy_check_mark: Use the provided logo sizes      |               :x: Violate the [CC BY-SA 4.0](https://creativecommons.org/licenses/by-sa/4.0/) license guidelines
+|                     |               :x: Mix/integrate your logo with our provided logos
+
+## 🖼 Logos
+
+<table>
+  <tr>
+    <td align="center" valign="middle"> <!-- Original version -->
+      <a href="https://github.com/devicons/devicon/blob/master/icons/devicon/devicon-original.svg" target="_blank">
+        <img width="150" src="https://raw.githubusercontent.com/devicons/devicon/master/icons/devicon/devicon-original.svg">
+
+**Original**
+
+---
+<a href="https://raw.githubusercontent.com/devicons/devicon/master/icons/devicon/devicon-original.svg">
+<svg aria-hidden="true" height="16" viewBox="0 0 16 16" version="1.1" width="16" data-view-component="true" class="octicon octicon-download mr-1">
+    <path fill-rule="evenodd" d="M7.47 10.78a.75.75 0 001.06 0l3.75-3.75a.75.75 0 00-1.06-1.06L8.75 8.44V1.75a.75.75 0 00-1.5 0v6.69L4.78 5.97a.75.75 0 00-1.06 1.06l3.75 3.75zM3.75 13a.75.75 0 000 1.5h8.5a.75.75 0 000-1.5h-8.5z"></path>
+</svg> Download
+</a>
+      </a>
+    </td> <!-- Plain version -->
+    <td align="center" valign="middle">
+      <a href="https://github.com/devicons/devicon/blob/master/icons/devicon/devicon-plain.svg" target="_blank">
+        <img width="150" src="https://raw.githubusercontent.com/devicons/devicon/master/icons/devicon/devicon-plain.svg">
+
+**Plain**
+
+---
+<a href="https://raw.githubusercontent.com/devicons/devicon/master/icons/devicon/devicon-plain.svg">
+<svg aria-hidden="true" height="16" viewBox="0 0 16 16" version="1.1" width="16" data-view-component="true" class="octicon octicon-download mr-1">
+    <path fill-rule="evenodd" d="M7.47 10.78a.75.75 0 001.06 0l3.75-3.75a.75.75 0 00-1.06-1.06L8.75 8.44V1.75a.75.75 0 00-1.5 0v6.69L4.78 5.97a.75.75 0 00-1.06 1.06l3.75 3.75zM3.75 13a.75.75 0 000 1.5h8.5a.75.75 0 000-1.5h-8.5z"></path>
+</svg> Download
+</a>
+    </a>
+  </td> <!-- Original wordmark version -->
+    <td align="center" valign="middle">
+      <a href="https://github.com/devicons/devicon/blob/master/icons/devicon/devicon-original-wordmark.svg" target="_blank">
+        <img width="150" src="https://raw.githubusercontent.com/devicons/devicon/master/icons/devicon/devicon-original-wordmark.svg">
+
+**Original Wordmark**
+
+---
+<a href="https://raw.githubusercontent.com/devicons/devicon/master/icons/devicon/devicon-original-wordmark.svg">
```

Comment:
I just noticed that our wordmark logo might need a redesign.. On white background it's impossible to see the wordmark.

![image](https://user-images.githubusercontent.com/8781699/140729608-4f6adf36-ba97-4a5a-a634-ba67de00f1af.png)

Reply:
> I just noticed that our wordmark logo might need a redesign.. On white background it's impossible to see the wordmark.
> 
> ![image](https://user-images.githubusercontent.com/8781699/140729608-4f6adf36-ba97-4a5a-a634-ba67de00f1af.png)

Maybe we should do that in another PR, but yes, you're right. 😄

Reply:
We could change the `original` version;s text to the `plain` version's text. Change the white color to green.